src/pages/SignIn/SignIn.tsx

```diff
@@ -3,33 +3,38 @@ import ForgotPasswordModal from "./ForgotPasswordModal";
 import LoginModal from "./LoginModal";
 import SignUpModal from "./SignUpModal";
 
-export default function SignIn() {
+export default function SignIn({ signUp }: { signUp: boolean }) {
   const openLogin = () => {
-    //@ts-ignore
     window.HSOverlay.open(document.querySelector("#login-modal"));
   };
 
   const openSignUp = () => {
-    // @ts-ignore
     window.HSOverlay.open(document.querySelector("#sign-up-modal"));
   };
 
   const openForgotPassword = () => {
-    // @ts-ignore
     window.HSOverlay.open(document.querySelector("#forgot-pass-modal"));
   };
 
   return (
-    <div className="m-8">
-      <Button
-        label="Get Started"
-        associatedModalId="sign-in-modal"
-        onClick={openSignUp}
-      />
-      <ForgotPasswordModal
-        id="forgot-pass-modal"
-        onLoginClick={openLogin}
-      />
+    <div>
+      {signUp ? (
+        <Button
+          label="Get Started"
+          associatedModalId="sign-in-modal"
+          onClick={openSignUp}
+          variation="FILLED"
+          className="bg-yellow"
+        />
+      ) : (
+        <Button
```

Comment:
This button should be the text variation, also the text needs to be bold (again you can use className prop for one time custom styling).

Reply:
outlines button looks better imo

Reply:
we need to follow the designers design. If you have a problem with their design please discuss during meeting or ping a designer on Discord and get approval. Otherwise follow the design. We have designers for a reason and they make choices like outlined vs text purposefully (or at least I would hope they do).